core/blocksigner/enclave.go

```diff
@@ -42,6 +43,12 @@ func (ec EnclaveClient) Sign(ctx context.Context, pk ed25519.PublicKey, bh *lega
 		client.BaseURL = tup[0]
 		client.AccessToken = tup[1]
 
+		// If the provided access token is just a password with no
+		// user, use a default basic auth username of 'chaincore'.
+		if !strings.Contains(client.AccessToken, ":") {
+			client.AccessToken = "chaincore:" + client.AccessToken
+		}
```

Comment:
instead of this business here, should rpc.Client prefer populating the basic auth password over the username? https://github.com/chain/chain/blob/main/core/rpc/rpc.go#L101-L111